### ISSUE TITLE
2025-03-26, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#121]
# Today's Arxiv Papers

(1) [CoLLM: A Large Language Model for Composed Image Retrieval](http://arxiv.org/abs/2503.19910v1), 2025-03-25

(2) [Scaling Vision Pre-Training to 4K Resolution](http://arxiv.org/abs/2503.19903v1), 2025-03-25

(3) [A Multi-Agent Framework Integrating Large Language Models and Generative AI for Accelerated Metamaterial Design](http://arxiv.org/abs/2503.19889v1), 2025-03-25

(4) [CausalRAG: Integrating Causal Graphs into Retrieval-Augmented Generation](http://arxiv.org/abs/2503.19878v1), 2025-03-25

(5) [SLA-Awareness for AI-assisted coding](http://arxiv.org/abs/2503.19876v1), 2025-03-25

(6) [Think Twice: Enhancing LLM Reasoning by Scaling Multi-round Test-time Thinking](http://arxiv.org/abs/2503.19855v1), 2025-03-25

(7) [FALCONEye: Finding Answers and Localizing Content in ONE-hour-long videos with multi-modal LLMs](http://arxiv.org/abs/2503.19850v1), 2025-03-25

(8) [Towards Online Multi-Modal Social Interaction Understanding](http://arxiv.org/abs/2503.19851v1), 2025-03-25

(9) [A Comparative Analysis of Word Segmentation, Part-of-Speech Tagging, and Named Entity Recognition for Historical Chinese Sources, 1900-1950](http://arxiv.org/abs/2503.19844v1), 2025-03-25

(10) [FireEdit: Fine-grained Instruction-based Image Editing via Region-aware Vision Language Model](http://arxiv.org/abs/2503.19839v1), 2025-03-25

(11) [SemEval-2025 Task 9: The Food Hazard Detection Challenge](http://arxiv.org/abs/2503.19800v1), 2025-03-25

(12) [PAVE: Patching and Adapting Video Large Language Models](http://arxiv.org/abs/2503.19794v1), 2025-03-25

(13) [ORION: A Holistic End-to-End Autonomous Driving Framework by Vision-Language Instructed Action Generation](http://arxiv.org/abs/2503.19755v1), 2025-03-25

(14) [Inducing Personality in LLM-Based Honeypot Agents: Measuring the Effect on Human-Like Agenda Generation](http://arxiv.org/abs/2503.19752v1), 2025-03-25

(15) [Optimizing Photonic Structures with Large Language Model Driven Algorithm Discovery](http://arxiv.org/abs/2503.19742v1), 2025-03-25

(16) [Writing as a testbed for open ended agents](http://arxiv.org/abs/2503.19711v1), 2025-03-25

(17) [AdaptiVocab: Enhancing LLM Efficiency in Focused Domains through Lightweight Vocabulary Adaptation](http://arxiv.org/abs/2503.19693v1), 2025-03-25

(18) [BiblioPage: A Dataset of Scanned Title Pages for Bibliographic Metadata Extraction](http://arxiv.org/abs/2503.19658v1), 2025-03-25

(19) [HausaNLP at SemEval-2025 Task 3: Towards a Fine-Grained Model-Aware Hallucination Detection](http://arxiv.org/abs/2503.19650v1), 2025-03-25

(20) [1.4 Million Open-Source Distilled Reasoning Dataset to Empower Large Language Model Training](http://arxiv.org/abs/2503.19633v1), 2025-03-25

(21) [Optimization through In-Context Learning and Iterative LLM Prompting for Nuclear Engineering Design Problems](http://arxiv.org/abs/2503.19620v1), 2025-03-25

(22) [Exploring Next Token Prediction For Optimizing Databases](http://arxiv.org/abs/2503.19619v1), 2025-03-25

(23) [RL-finetuning LLMs from on- and off-policy data with a single algorithm](http://arxiv.org/abs/2503.19612v1), 2025-03-25

(24) [Innate Reasoning is Not Enough: In-Context Learning Enhances Reasoning Large Language Models with Less Overthinking](http://arxiv.org/abs/2503.19602v1), 2025-03-25

(25) [HoarePrompt: Structural Reasoning About Program Correctness in Natural Language](http://arxiv.org/abs/2503.19599v1), 2025-03-25

(26) [The Greatest Good Benchmark: Measuring LLMs' Alignment with Utilitarian Moral Dilemmas](http://arxiv.org/abs/2503.19598v1), 2025-03-25

(27) [Context-Efficient Retrieval with Factual Decomposition](http://arxiv.org/abs/2503.19574v1), 2025-03-25

(28) [Motif Counting in Complex Networks: A Comprehensive Survey](http://arxiv.org/abs/2503.19573v1), 2025-03-25

(29) [Scaling Laws of Synthetic Data for Language Models](http://arxiv.org/abs/2503.19551v1), 2025-03-25

(30) [FLEX: A Benchmark for Evaluating Robustness of Fairness in Large Language Models](http://arxiv.org/abs/2503.19540v1), 2025-03-25

(31) [Agent-Initiated Interaction in Phone UI Automation](http://arxiv.org/abs/2503.19537v1), 2025-03-25

(32) [DomainCQA: Crafting Expert-Level QA from Domain-Specific Charts](http://arxiv.org/abs/2503.19498v1), 2025-03-25

(33) [KSHSeek: Data-Driven Approaches to Mitigating and Detecting Knowledge-Shortcut Hallucinations in Generative Models](http://arxiv.org/abs/2503.19482v1), 2025-03-25

(34) [GenHancer: Imperfect Generative Models are Secretly Strong Vision-Centric Enhancers](http://arxiv.org/abs/2503.19480v1), 2025-03-25

(35) [A-MESS: Anchor based Multimodal Embedding with Semantic Synchronization for Multimodal Intent Recognition](http://arxiv.org/abs/2503.19474v1), 2025-03-25

(36) [ReSearch: Learning to Reason with Search for LLMs via Reinforcement Learning](http://arxiv.org/abs/2503.19470v1), 2025-03-25

(37) [Data-centric Federated Graph Learning with Large Language Models](http://arxiv.org/abs/2503.19455v1), 2025-03-25

(38) [VecTrans: LLM Transformation Framework for Better Auto-vectorization on High-performance CPU](http://arxiv.org/abs/2503.19449v1), 2025-03-25

(39) [Enhanced Bloom's Educational Taxonomy for Fostering Information Literacy in the Era of Large Language Models](http://arxiv.org/abs/2503.19434v1), 2025-03-25

(40) [DeCAP: Context-Adaptive Prompt Generation for Debiasing Zero-shot Question Answering in Large Language Models](http://arxiv.org/abs/2503.19426v1), 2025-03-25

(41) [LangBridge: Interpreting Image as a Combination of Language Embeddings](http://arxiv.org/abs/2503.19404v1), 2025-03-25

(42) [Inference-Time Scaling for Flow Models via Stochastic Generation and Rollover Budget Forcing](http://arxiv.org/abs/2503.19385v1), 2025-03-25

(43) [QUAD: Quantization and Parameter-Efficient Tuning of LLM with Activation Decomposition](http://arxiv.org/abs/2503.19353v1), 2025-03-25

(44) [Multi-Object Sketch Animation by Scene Decomposition and Motion Planning](http://arxiv.org/abs/2503.19351v1), 2025-03-25

(45) [Membership Inference Attacks on Large-Scale Models: A Survey](http://arxiv.org/abs/2503.19338v1), 2025-03-25

(46) [Process or Result? Manipulated Ending Tokens Can Mislead Reasoning LLMs to Ignore the Correct Reasoning Steps](http://arxiv.org/abs/2503.19326v1), 2025-03-25

(47) [ImageGen-CoT: Enhancing Text-to-Image In-context Learning with Chain-of-Thought Reasoning](http://arxiv.org/abs/2503.19312v1), 2025-03-25

(48) [LRSCLIP: A Vision-Language Foundation Model for Aligning Remote Sensing Image with Longer Text](http://arxiv.org/abs/2503.19311v1), 2025-03-25

(49) [Iterative Hypothesis Generation for Scientific Discovery with Monte Carlo Nash Equilibrium Self-Refining Trees](http://arxiv.org/abs/2503.19309v1), 2025-03-25

(50) [Context-Aware Semantic Segmentation: Enhancing Pixel-Level Understanding with Large Language Models for Advanced Vision Applications](http://arxiv.org/abs/2503.19276v1), 2025-03-25

(51) [MARS: Memory-Enhanced Agents with Reflective Self-improvement](http://arxiv.org/abs/2503.19271v1), 2025-03-25

(52) [PHEONA: An Evaluation Framework for Large Language Model-based Approaches to Computational Phenotyping](http://arxiv.org/abs/2503.19265v1), 2025-03-25

(53) [DWIM: Towards Tool-aware Visual Reasoning via Discrepancy-aware Workflow Generation & Instruct-Masking Tuning](http://arxiv.org/abs/2503.19263v1), 2025-03-25

(54) [Linguistic Blind Spots of Large Language Models](http://arxiv.org/abs/2503.19260v1), 2025-03-25

(55) [SCI-IDEA: Context-Aware Scientific Ideation Using Token and Sentence Embeddings](http://arxiv.org/abs/2503.19257v1), 2025-03-25

(56) [LLM Benchmarking with LLaMA2: Evaluating Code Development Performance Across Multiple Programming Languages](http://arxiv.org/abs/2503.19217v1), 2025-03-24

(57) [A Survey of Large Language Model Agents for Question Answering](http://arxiv.org/abs/2503.19213v1), 2025-03-24

(58) [Overtrained Language Models Are Harder to Fine-Tune](http://arxiv.org/abs/2503.19206v1), 2025-03-24

(59) [A Shared Low-Rank Adaptation Approach to Personalized RLHF](http://arxiv.org/abs/2503.19201v1), 2025-03-24

(60) [Open-Vocabulary Functional 3D Scene Graphs for Real-World Indoor Spaces](http://arxiv.org/abs/2503.19199v1), 2025-03-24

(61) [Evaluating Bias in LLMs for Job-Resume Matching: Gender, Race, and Education](http://arxiv.org/abs/2503.19182v1), 2025-03-24

(62) [AssertionForge: Enhancing Formal Verification Assertion Generation with Structured Representation of Specifications and RTL](http://arxiv.org/abs/2503.19174v1), 2025-03-24

(63) [Language Model Uncertainty Quantification with Attention Chain](http://arxiv.org/abs/2503.19168v1), 2025-03-24

(64) [HOIGPT: Learning Long Sequence Hand-Object Interaction with Language Models](http://arxiv.org/abs/2503.19157v1), 2025-03-24

(65) [Compositional Caching for Training-free Open-vocabulary Attribute Detection](http://arxiv.org/abs/2503.19145v1), 2025-03-24

(66) [MIRAGE: Multimodal Immersive Reasoning and Guided Exploration for Red-Team Jailbreak Attacks](http://arxiv.org/abs/2503.19134v1), 2025-03-24

(67) [Understanding and Improving Information Preservation in Prompt Compression for LLMs](http://arxiv.org/abs/2503.19114v1), 2025-03-24

(68) [Masks and Mimicry: Strategic Obfuscation and Impersonation Attacks on Authorship Verification](http://arxiv.org/abs/2503.19099v1), 2025-03-24

(69) [Rankers, Judges, and Assistants: Towards Understanding the Interplay of LLMs in Information Retrieval Evaluation](http://arxiv.org/abs/2503.19092v1), 2025-03-24

(70) [LLM-Based Insight Extraction for Contact Center Analytics and Cost-Efficient Deployment](http://arxiv.org/abs/2503.19090v1), 2025-03-24

(71) [Mist: Efficient Distributed Training of Large Language Models via Memory-Parallelism Co-Optimization](http://dx.doi.org/10.1145/3689031.3717461), 2025-03-24

(72) [LookAhead Tuning: Safer Language Models via Partial Answer Previews](http://arxiv.org/abs/2503.19041v1), 2025-03-24

(73) [SlowFast-LLaVA-1.5: A Family of Token-Efficient Video Large Language Models for Long-Form Video Understanding](http://arxiv.org/abs/2503.18943v1), 2025-03-24

(74) [Video-T1: Test-Time Scaling for Video Generation](http://arxiv.org/abs/2503.18942v1), 2025-03-24

(75) [Exploring Training and Inference Scaling Laws in Generative Retrieval](http://arxiv.org/abs/2503.18941v1), 2025-03-24

(76) [CoMP: Continual Multimodal Pre-training for Vision Foundation Models](http://arxiv.org/abs/2503.18931v1), 2025-03-24

(77) [Trajectory Balance with Asynchrony: Decoupling Exploration and Learning for Fast, Scalable LLM Post-Training](http://arxiv.org/abs/2503.18929v1), 2025-03-24

(78) [Video SimpleQA: Towards Factuality Evaluation in Large Video Language Models](http://arxiv.org/abs/2503.18923v1), 2025-03-24

(79) [FFN Fusion: Rethinking Sequential Computation in Large Language Models](http://arxiv.org/abs/2503.18908v1), 2025-03-24

(80) [xKV: Cross-Layer SVD for KV-Cache Compression](http://arxiv.org/abs/2503.18893v1), 2025-03-24

(81) [AgentDropout: Dynamic Agent Elimination for Token-Efficient and High-Performance LLM-Based Multi-Agent Collaboration](http://arxiv.org/abs/2503.18891v1), 2025-03-24

(82) [Toward building next-generation Geocoding systems: a systematic review](http://arxiv.org/abs/2503.18888v1), 2025-03-24

(83) [I Have Covered All the Bases Here: Interpreting Reasoning Features in Large Language Models via Sparse Autoencoders](http://arxiv.org/abs/2503.18878v1), 2025-03-24

(84) [Reimagining Memory Access for LLM Inference: Compression-Aware Memory Controller Design](http://arxiv.org/abs/2503.18869v1), 2025-03-24

(85) [Structuring Scientific Innovation: A Framework for Modeling and Discovering Impactful Knowledge Combinations](http://arxiv.org/abs/2503.18865v2), 2025-03-24

(86) [EconEvals: Benchmarks and Litmus Tests for LLM Agents in Unknown Environments](http://arxiv.org/abs/2503.18825v1), 2025-03-24

(87) [Defeating Prompt Injections by Design](http://arxiv.org/abs/2503.18813v1), 2025-03-24

(88) [Classical Planning with LLM-Generated Heuristics: Challenging the State of the Art with Python Code](http://arxiv.org/abs/2503.18809v1), 2025-03-24

(89) [Option Discovery Using LLM-guided Semantic Hierarchical Reinforcement Learning](http://arxiv.org/abs/2503.19007v1), 2025-03-24

(90) [REALM: A Dataset of Real-World LLM Use Cases](http://arxiv.org/abs/2503.18792v1), 2025-03-24

(91) [BitDecoding: Unlocking Tensor Cores for Long-Context LLMs Decoding with Low-Bit KV Cache](http://arxiv.org/abs/2503.18773v1), 2025-03-24

(92) [AlphaSpace: Enabling Robotic Actions through Semantic Tokenization and Symbolic Reasoning](http://arxiv.org/abs/2503.18769v1), 2025-03-24

(93) [Synthetic Function Demonstrations Improve Generation in Low-Resource Programming Languages](http://arxiv.org/abs/2503.18760v1), 2025-03-24

(94) [The Quantum Technology Job Market: A Quantitative Investigation](http://arxiv.org/abs/2503.19004v1), 2025-03-24

(95) [LLaVAction: evaluating and training multi-modal large language models for action recognition](http://arxiv.org/abs/2503.18712v1), 2025-03-24

(96) [Commander-GPT: Fully Unleashing the Sarcasm Detection Capability of Multi-Modal Large Language Models](http://arxiv.org/abs/2503.18681v2), 2025-03-24

(97) [AgentSpec: Customizable Runtime Enforcement for Safe and Reliable LLM Agents](http://arxiv.org/abs/2503.18666v1), 2025-03-24

(98) [Boosting Virtual Agent Learning and Reasoning: A Step-wise, Multi-dimensional, and Generalist Reward Model with Benchmark](http://arxiv.org/abs/2503.18665v1), 2025-03-24

(99) [From Fragment to One Piece: A Survey on AI-Driven Graphic Design](http://arxiv.org/abs/2503.18641v1), 2025-03-24

(100) [Unbiasing through Textual Descriptions: Mitigating Representation Bias in Video Benchmarks](http://arxiv.org/abs/2503.18637v1), 2025-03-24
